editor/test/integration/save_data_test.clj

```diff
@@ -1332,7 +1332,7 @@
   (test-util/with-loaded-project project-path
     (test-util/clear-cached-save-data! project)
     (doseq [save-data (project->save-datas project)]
-      (test-util/check-save-data-disk-equivalence! save-data))))
+      (test-util/check-save-data-disk-equivalence! save-data project-path))))
```

Comment:
Fix: The message on test failure would report the wrong path to the resources on disk.